services/gateway/src/middleware/request-logger.ts

```diff
@@ -1,5 +1,6 @@
 import * as _ from "lodash";
 import * as morgan from "morgan";
+import * as qs from "qs";
```

Comment:
Can't we use the built-in `querystring` module?

Reply:
No, it doesn't work with nested object.